engine/lite/components/kafka-tests/src/test/scala/pl/touk/nussknacker/engine/lite/components/LiteKafkaUniversalJsonFunctionalTest.scala

```diff
@@ -286,10 +286,12 @@ class LiteKafkaUniversalJsonFunctionalTest
       config(Json.obj(), schemaObjStr, schemaEnumStrOrList, output = SpecialSpELElement("{1,2}"), lax.headOption)
     val results = runWithValueResults(cfg)
 
-    results.isValid shouldBe true // it should be invalid, but it's so edge case that we decided to live with it
-    val runtimeError = results.validValue.errors.head
-    runtimeError.nodeId shouldBe Some("my-sink")
-    runtimeError.throwable.asInstanceOf[RuntimeException].getMessage shouldBe "#: [1,2] is not a valid enum value"
+    results.isValid shouldBe false
+    val runtimeError = results.invalidValue.head
+    runtimeError.nodeIds shouldBe Set("my-sink")
+    runtimeError.asInstanceOf[CustomNodeError].message shouldBe
+      """Provided value does not match scenario output - errors:
+        |Incorrect type: actual: 'List[Integer]([1, 2])' expected: 'List[Integer]([1, 2, 3]) | String(A)'.""".stripMargin
```

Comment:
I think that we shold rather use spel notation to display collection types - `List[Integer]({1, 2, 3})`